makefiles/features_modules.inc.mk

```diff
@@ -16,8 +16,13 @@ ifneq (,$(filter periph_init, $(USEMODULE)))
     periph_common \
     periph_flexcomm \
     periph_gpio_mux \
+    periph_i2c_hw \
```

Comment:
Is this something we need in general or just specifically for ESPs?

Reply:
Well, the list is global, but these modules appear only for ESPs. It is there just to avoid unnecessary `periph_init_` modules automatically derived.